src/node_modules/app/elasticsearch/config.spec.js

```diff
@@ -0,0 +1,48 @@
+import assert from 'assert';
+import * as _ from 'lamb';
+import { docsByParamByAgg, paramsByAggs } from './config';
+
+const evaluate = _.when(_.isType('Function'), _.applyTo(['dummyType']));
+
+const process = _.pipe([
+	_.pairs,
+	_.mapWith(([key, definition]) => [key, evaluate(definition)]),
+]);
```

Comment:
here we can use `valuesWith`:

```suggestion
const process = valuesWith((definition, key) => [key, evaluate(definition)]);
```

Also probably I wouldn't use `process` to avoid shadowing `node`'s global (which reminds me I had a note to rename `process` to `run` in `make_data`)